templates/src/pages/Home.js

```diff
@@ -14,32 +13,51 @@ function Home() {
   const [status, setStatus] = useState({
     code: 'Checking...',
   })
+<%if eq (index .Params `userAuth`) "yes" %>
+  const [privateState, setPrivateData] = useState({
+    isLoading: true,
+    data: {},
+  });
+
+  const fetchPrivateData = async(data) =>  {
+    const uri = `${config.backendURL}/api/auth-data`
```

Comment:
See path change requests in https://github.com/commitdev/zero-deployable-node-backend/pull/10